lib/ClientFactory.php

```diff
@@ -9,10 +9,12 @@
   * @link      http://packagist.org/packages/sendgrid/sendgrid
   */
 
+namespace SendGrid;
+
 /**
   * Interface to the SendGrid Web API
   */
-class SendGrid
+class ClientFactory
```

Comment:
Since we are not calling this a Factory, what do you think about returning the client rather then just setting the client in this class?

Also, I'm thinking of adding a function `send` to this class, where we would take in a SendGridMessage object and return $response. If we returned the client, then we would also need to pass in a client to the `send` function. The advantage of that is then we can allow for custom clients to be passed in. 

Just thinking out loud here, wanting to make sure we get this interface right since I'm hoping to not make another breaking change in a long time :)